### PR TITLE
chore: fix readme links

### PR DESCRIPTION
**Pull request by JuroUhlar**

### CODE REVIEW

**Review by necipallef — APPROVED** [review]

**Review by Orkuncakilkaya — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -256,14 +256,14 @@ This is how fields in the Page Spec are populated:
...
-| `context.confidenceScore` | [`result.confidenceScore`](https://dev.fingerprint.com/docs/js-agent#confidence) - Confidence score                                      |
-| `context.requestId`       | [`result.requestId`](https://dev.fingerprint.com/docs/js-agent#requestid) - ID of Fingerprint request                                    |
-| `properties.url`          | [`result.url`](https://dev.fingerprint.com/docs/webhooks#identification-webhook-object-format) - page URL                                |
+| `context.ip`              | [`result.ip`](https://dev.fingerprint.com/reference/get-function#ip) - IP address                                                                 |
+| `context.browserDetails`  | [`result.browserDetails`](https://dev.fingerprint.com/docs/webhooks#json-example) - Browser, OS name and version |
+| `context.incognito`       | [`result.incognito`](https://dev.fingerprint.com/reference/get-function#incognito) - Incognito mode                                               |
+| `context.confidenceScore` | [`result.confidenceScore`](https://dev.fingerprint.com/reference/get-function#confidence) - Confidence score                                      |
```
Comment (Orkuncakilkaya) [inline README.md:264]: Is the text inside documentation for confidence, updated with our new Confidence score v1.1?

File: README.md [hunk 2]
```diff
@@ -224,9 +224,9 @@ This is how fields in the Identify Spec are populated:
 
 | Identify Spec field | Source field in Fingerprint Pro `result` or metadata passed to `segment.identify`                                  |
 | ------------------- | ------------------------------------------------------------------------------------------------------------------ |
-| `anonymousId`       | [`result.visitorId`](https://dev.fingerprint.com/docs/js-agent#visitorid) - Fingerprint `visitorId`                |
-| `traits.visitorId`  | [`result.visitorId`](https://dev.fingerprint.com/docs/js-agent#visitorid) - Fingerprint `visitorId`                |
-| `traits.createdAt`  | [`result.firstSeenAt.subscription`](https://dev.fingerprint.com/docs/js-agent#firstseenat) - time of first visit   |
+| `anonymousId`       | [`result.visitorId`](https://dev.fingerprint.com/reference/get-function#visitorid) - Fingerprint `visitorId`                |
+| `traits.visitorId`  | [`result.visitorId`](https://dev.fingerprint.com/reference/get-function#visitorid) - Fingerprint `visitorId`                |
+| `traits.createdAt`  | [`result.firstSeenAt.subscription`](https://dev.fingerprint.com/reference/get-function#firstseenat) - time of first visit   |
```
Comment (Orkuncakilkaya) [inline README.md:229]: Is the text inside documentation for firstSeenAt updated after our DRN product? I believe DRN changed how firstSeenAt's value

File: README.md (hunk 1 above)
Comment (JuroUhlar) [reply README.md:264]: Yes!

**Comment by JuroUhlar** [comment]
@Orkuncakilkaya if the threads are resolved, can you please merge this? I don't know how to run or bypass the pipelines. Thank you!

**Pull request merged**